frontend/src/metabase/home/homepage/components/HomeGreeting/HomeGreeting.tsx

```diff
@@ -36,12 +36,13 @@ const HomeGreeting = ({
 };
 
 const getMessage = (name: string): string => {
+  const namePart = name ? `, ${name}` : "";
   const options = [
-    t`Hey there, ${name}`,
-    t`How's it going, ${name}?`,
-    t`Howdy, ${name}`,
-    t`Greetings, ${name}`,
-    t`Good to see you, ${name}`,
+    t`Hey there` + namePart,
```

Comment:
It looks to me like both in the current change as well as in Alexander's suggestion there's a missing space between "there" and the namePart.

Reply:
@mazameli That's because `namePart` could contain an optional comma so a single translation could result in 2 separate greetings
1. "Hey there": When names are null
2. "Hey there, Maz"

Reply:
Gotcha gotcha 👌🏼